docs/data/joy/components/accordion/accordion.md

```diff
@@ -0,0 +1,50 @@
+---
+product: joy-ui
+title: React Accordion component
+githubLabel: 'component: accordion'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+---
+
+# Accordion
+
+<p class="description">The accordion component allows the user to show and hide sections of related content on a page.</p>
+
+:::success
+Joy Accordion component is still **in development** but it should not block you from using Joy UI!
+
+The examples below demonstrate the featuring between Joy UI and popular headless libraries.
```

Comment:
```suggestion
This document shows how construct an accordion with existing Joy UI components and popular headless UI libraries.
```

Reply:
Do you miss `to` between `how … construct`?